flaml/nlp/huggingface/training_args.py

```diff
@@ -0,0 +1,139 @@
+import argparse
+from dataclasses import dataclass, field
+
+from ...data import (
+    NLG_TASKS,
+)
+from typing import Optional, List
+
+try:
+    from transformers import TrainingArguments, Seq2SeqTrainingArguments
+except ImportError:
+    Seq2SeqTrainer = object
+
+from transformers import IntervalStrategy
```

Comment:
Shouldn't this be included in try...except?

Reply:
Removed since not used in the other files.